glue/viewers/matplotlib/state.py

```diff
@@ -273,10 +273,11 @@ class MatplotlibLayerState(LayerState):
 
     def __init__(self, viewer_state=None, **kwargs):
 
-        super(MatplotlibLayerState, self).__init__(viewer_state=viewer_state, **kwargs)
+        super(MatplotlibLayerState, self).__init__(viewer_state=viewer_state, preferred_cmap=None, **kwargs)
 
         self.color = self.layer.style.color
         self.alpha = self.layer.style.alpha
+        self.preferred_cmap = self.layer.style.preferred_cmap
```

Comment:
I think the changes in this file are no longer needed

Reply:
Done